executor/executor.go

```diff
@@ -979,7 +980,7 @@ func newLockCtx(seVars *variable.SessionVars, lockWaitTime int64) *tikvstore.Loc
 	lockCtx.LockKeysDuration = &seVars.StmtCtx.LockKeysDuration
 	lockCtx.LockKeysCount = &seVars.StmtCtx.LockKeysCount
 	lockCtx.LockExpired = &seVars.TxnCtx.LockExpire
-	lockCtx.ResourceGroupTag = resourcegrouptag.EncodeResourceGroupTag(sqlDigest, planDigest)
+	lockCtx.ResourceGroupTag = resourcegrouptag.EncodeResourceGroupTag(sqlDigest, planDigest, tipb.ResourceGroupTagLabel_ResourceGroupTagLabelUnknown)
```

Comment:
Why not also use `SetResourceGroupTagger` too?

Reply:
It seems to be missed here, but does it make sense to distinguish row/index on `PessimisticLockRequest`? If so, I will push an update.

Reply:
Yes, it makes sense.

Reply:
https://github.com/tikv/client-go/pull/381
PTAL, thanks. I will update the code of this PR after the PR of `client-go` is merged.
cc @zhongzc @crazycs520

Reply:
https://github.com/tikv/client-go/pull/381
PTAL, thanks.  I will update this PR after the PR of `client-go` is merged.
cc @zhongzc @crazycs520

Reply:
done.

Reply:
https://github.com/pingcap/tidb/pull/29044/files#diff-4c9a4c1e9a402d07a0a78f4db9f766d23f1ff57ceda1e75c4e83b579488ad301R978